### PR TITLE
Add option to show the time modification of folders\files

### PR DESCRIPTION
**Pull request by mk410gd**
In Settings  has been added option to show the time modification of folders\files in the list of items (issue #1457)

### CODE REVIEW

File: app/src/main/java/com/amaze/filemanager/adapters/data/LayoutElementParcelable.java [hunk 1]
```diff
@@ -141,10 +141,10 @@ public LayoutElementParcelable(Parcel im) {
         symlink = im.readString();
         int j = im.readInt();
         date = im.readLong();
+        dateModification = im.readString();
```
Comment (EmmanuelMess) [inline app/src/main/java/com/amaze/filemanager/adapters/data/LayoutElementParcelable.java]: This line is in the wrong position, I think, the `Parcel` must be read in the same order it is written.

File: app/src/main/res/xml/preferences.xml [hunk 2]
```diff
@@ -84,7 +84,7 @@
             android:key="showHidden"
             android:title="@string/hidden"/>
         <com.amaze.filemanager.ui.views.preference.CheckBox
-            android:defaultValue="true"
+            android:defaultValue="false"
```
Comment (EmmanuelMess) [inline app/src/main/res/xml/preferences.xml]: Why did you change the default value?

Comment (mk410gd) [reply app/src/main/res/xml/preferences.xml]: my fault, default value was changed

**Comment by mk410gd** [comment]
@EmmanuelMess added the same functionality to the compressed files
